draft-ietf-scitt-architecture.md

```diff
@@ -102,9 +102,9 @@ Producers can register their Signed Statements on any Transparency Service, with
 This document describes a scalable and flexible, decentralized architecture to enhance auditability and accountability across various existing and emerging supply chains.
 It achieves this goal by enforcing the following complementary security guarantees:
 
-1. Statements made by Issuers about supply chain Artifacts must be identifiable, authentic, and non-repudiable;
-2. such Statements must be registered on a secure append-only Log, so that their provenance and history can be independently and consistently audited;
-3. Issuers can efficiently prove to any other party the Registration of their Signed Statements; verifying this proof ensures that the Issuer is consistent and non-equivocal when producing Signed Statements.
+1. Statements made by Issuers about supply chain Artifacts must be identifiable, authentic, and non-repudiable
+1. Such Statements must be registered on a secure append-only Log, so that their provenance and history can be independently and consistently audited
+1. Issuers can efficiently prove to any other party the Registration of their Signed Statements; verifying this proof ensures that the Issuer is consistent and non-equivocal when producing Signed Statements
```

Comment:
```suggestion
2. Such Statements must be registered on a secure append-only Log, so that their provenance and history can be independently and consistently audited
3. Issuers can efficiently prove to any other party the Registration of their Signed Statements; verifying this proof ensures that the Issuer is consistent and non-equivocal when producing Signed Statements
```